index.css

```diff
@@ -0,0 +1,123 @@
+body {
+  margin: 0;
+  padding: 0;
+}
+
+.page1 {
+  height: 100vh;
+  background-color: #1c1a19;
+}
+
+header {
+  position: absolute;
+  width: 100%;
+  top: 0;
+  display: flex;
+  background: #3e3c3c;
+  justify-content: space-between;
+  align-items: baseline;
+}
+
+header h1 {
+  margin: 3%;
+}
+
+header h1 a {
+  text-decoration: none;
+  color: #fff;
+}
+
+header ul {
+  display: flex;
+  justify-content: flex-end;
+}
+
+header li {
+  list-style: none;
+  width: 100px;
+}
+
+header li a {
+  padding: 10px;
+  text-decoration: none;
+  color: #fff;
+}
+
+header li a:hover {
+  background: #b3adad;
+}
+
+aside {
+  position: absolute;
+  margin-top: 30%;
+  width: 40px;
+}
+
+aside ul {
+  display: flex;
+  flex-direction: column;
+  flex-wrap: wrap;
+  align-items: center;
+}
+
+aside ul li {
+  list-style: none;
+  height: 100px;
+}
+
+aside ul li i {
+  color: #cfcfcf;
+}
+
+aside ul li a:hover i {
+  color: rgb(248, 176, 245);
+}
+
+.home {
+  position: absolute;
+  display: flex;
+  top: 30vh;
+  left: 15%;
+}
+
+.home .text {
+  flex: 1;
+}
+
+.home .image {
+  flex: 1;
+  display: flex;
+  flex-wrap: wrap;
+  justify-content: flex-end;
+}
+
+.home .text h2 {
+  font-family: Crete Round;
```

Comment:
- [ ] I think this is the reason why your stylint linter is failing.
 `Crete Round` is an external font. Therefore you need to associate a generic font with it.  And `serif` is a common generic font family.

 Try this out `font-family: 'Crete Round', serif;`

Please try the above in all lines where you used external fonts.